sql/database.sql

```diff
@@ -2502,6 +2502,7 @@ CREATE TABLE `insurance_companies` (
   `x12_receiver_id` varchar(25) default NULL,
   `x12_default_partner_id` int(11) default NULL,
   `alt_cms_id` varchar(15) NOT NULL DEFAULT '',
+  `inactive` int(1) NOT NULL,
   PRIMARY KEY  (`id`)
 ) ENGINE=InnoDB;
```

Comment:
Also need to add this entry in the sql upgrade script at the bottom of 4_2_2-to-5_0_0_upgrade.sql:
``````
#IfMissingColumn insurance_companies inactive
	ALTER TABLE `insurance_companies` ADD `inactive` int(1) NOT NULL DEFAULT 0;
#EndIf